### ISSUE TITLE
versatile diffusion reconstruction is worse than reported

### ISSUE THREAD

**Issue by cnwangy** [ozcelikfu/brain-diffuser#9]
I followed the instruction to run the code. The first stage vdvae results are OK and the evalution metrics are the same as in the paper. However, the second stage of versatile diffusion reconstruction is worse that in the paper. Can you help to locate where is the problem? Below are two examples.
![70](https://github.com/ozcelikfu/brain-diffuser/assets/13940405/719f6e4d-73a9-4218-b740-6e7e19245ee5)
![70](https://github.com/ozcelikfu/brain-diffuser/assets/13940405/90173c3b-8a9b-4215-bdec-8b46c78fabc0)

![216](https://github.com/ozcelikfu/brain-diffuser/assets/13940405/ef9d38ab-9300-45f9-9a51-5b6531234ead)
![216](https://github.com/ozcelikfu/brain-diffuser/assets/13940405/04c82c67-cc08-45a3-b101-c4e31fa6908d)

**Comment by MRLQ-Q** [comment]
May I ask how the original image is displayed? I ran it according to the code and only had the result image, so I couldn't see the original image. Looking forward to your guidance. Thank you.

**Comment by ozcelikfu** [comment]
@cnwangy Well, I never encountered any results like these but it seems the diffusion model doesn't seem to working properly, have you changed any parameters in the versatilediffusion_reconstruct_images.py like strength or mixing?

**Comment by MRLQ-Q** [comment]
I also followed the steps you mentioned for training, and the training results were the same as his, very blurry, like partial screenshots of the original image.

**Comment by cnwangy** [comment]
I think I have solved the problem. It is the version of the transformers lib. When I use v4.33.2 the extracted clipvision features are different from v4.19.2. Using the latter one I can replicate your results. Thanks a lot!

**Comment by MRLQ-Q** [comment]
> I think I have solved the problem. It is the version of the transformers lib. When I use v4.33.2 the extracted clipvision features are different from v4.19.2. Using the latter one I can replicate your results. Thanks a lot!

May I ask how you solved it and how to use v4.33.2? Can you provide me with a detailed explanation? Because I am a novice, please forgive me. Thank you.

**Comment by cnwangy** [comment]
> > I think I have solved the problem. It is the version of the transformers lib. When I use v4.33.2 the extracted clipvision features are different from v4.19.2. Using the latter one I can replicate your results. Thanks a lot!
> 
> May I ask how you solved it and how to use v4.33.2? Can you provide me with a detailed explanation? Because I am a novice, please forgive me. Thank you.

You can just follow the README to use transformers==v4.19.2 as in the environment.yaml. Do not upgrade to v4.33.2.

**Comment by MRLQ-Q** [comment]
> > > I think I have solved the problem. It is the version of the transformers lib. When I use v4.33.2 the extracted clipvision features are different from v4.19.2. Using the latter one I can replicate your results. Thanks a lot!
> > 
> > 
> > May I ask how you solved it and how to use v4.33.2? Can you provide me with a detailed explanation? Because I am a novice, please forgive me. Thank you.
> 
> You can just follow the README to use transformers==v4.19.2 as in the environment.yaml. Do not upgrade to v4.33.2.

Thank you very much for your guidance. I have successfully reproduced it.

**Comment by happylufei** [comment]
> > > I think I have solved the problem. It is the version of the transformers lib. When I use v4.33.2 the extracted clipvision features are different from v4.19.2. Using the latter one I can replicate your results. Thanks a lot!
> > 
> > 
> > May I ask how you solved it and how to use v4.33.2? Can you provide me with a detailed explanation? Because I am a novice, please forgive me. Thank you.
> 
> You can just follow the README to use transformers==v4.19.2 as in the environment.yaml. Do not upgrade to v4.33.2.

Thank you very much for the solution you provided. However, I encountered the following error after downgrading the transforms library.
`'(MaxRetryError("HTTPSConnectionPool(host='huggingface.co', port=443): Max retries exceeded with url: /openai/clip-vit-large-patch14/resolve/main/vocab.json (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f1af21956d0>, 'Connection to huggingface.co timed out. (connect timeout=10)'))"), '(Request ID: 3df39c48-06cc-45e7-8096-22b9ca4c9f91)')' thrown while requesting HEAD https://huggingface.co/openai/clip-vit-large-patch14/resolve/main/**vocab.json**`
 I've tried to resolve it, but it seems that Hugging Face no longer provides older versions of the models. I'm not sure if you've encountered the same issue or if you have any suggestions for a solution. Once again, thank you for your help
